internal/provider/resource_ably_key.go

```diff
@@ -48,6 +48,14 @@ func (r resourceKey) GetSchema(_ context.Context) (tfsdk.Schema, diag.Diagnostic
 				Required:    true,
 				Description: "The capabilities that this key has. More information on capabilities can be found in the [Ably documentation](https://ably.com/docs/core-features/authentication#capabilities-explained)",
 			},
+			"revocable_tokens": {
+				Type:        types.BoolType,
+				Required:    true,
+				Description: "Enable Token Revocation on the Ably Key. More information on Token Revocation can be found in the [Ably documentation](https://ably.com/docs/auth/revocation)",
```

Comment:
I think this makes it slightly more obvious what the option does without needing to consult the docs
```suggestion
				Description: "Allow tokens issued by this key to be revoked. More information on Token Revocation can be found in the [Ably documentation](https://ably.com/docs/auth/revocation)",
```

Reply:
Modified. Thanks for the suggestion!